vetter2020daily/Vetter2020Daily.yaml

```diff
@@ -0,0 +1,497 @@
+# yaml-language-server: $schema=.schema.yaml
+title: 'Daily Viral Kinetics and Innate and Adaptive Immune Response Assessment in
+  COVID-19: a Case Series'
+doi: https://doi.org/10.1128/msphere.00827-20
+description: >-
```

Comment:
For `folded` block, to remove the `-` after `>`. You need to add "\n" at the end of your string within the block. I will add another comment in `vetter2020daily.py.